src/bin/inx-chronicle/api/stardust/analytics/responses.rs

```diff
@@ -102,3 +102,22 @@ impl From<DistributionStat> for DistributionStatDto {
         }
     }
 }
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MilestoneStatsResponse {
+    pub blocks: usize,
+    pub per_payload_type: MilestoneStatsPerPayloadTypeDto,
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MilestoneStatsPerPayloadTypeDto {
```

Comment:
Why don't we return all the statistics that we have, like the number of treasury transactions?

Reply:
No not at the moment because it wasnt requested.